DOC-6330 Note features now available in CE (#1126)
couchbase/docs-server: modules/release-notes/pages/relnotes.adoc

Starting with this release, the use of VIEW indexes is deprecated in N1QL only.
Node-to-node encryption is unsupported for the use of VIEW indexes in N1QL only.

> ==== Features now available in Couchbase Server Community Edition

Reviewer: ```suggestion
==== Features Now Available in Couchbase Server Community Edition
```

> Starting with this release, the following features are available in the Community Edition of Couchbase Server as well:

Reviewer: ```suggestion
Starting with this release, the following features have been made available in the Community Edition of Couchbase Server:
```
